steps/kubernetes.py

```diff
@@ -0,0 +1,200 @@
+'''test methods for kubernetes'''
+
+import re
+import json
+import time
+from jinja2 import Environment, FileSystemLoader
+from behave import *
+
+# NB: Note that we aggregate across masters. This doesn't matter in reality
+# since our cluster test uses only one master. But if we ever change to using
+# multiple masters, things will be different.
+
+def kubectl_get(context, component):
+    r = context.remote_cmd('command',
+                           module_args='kubectl get %s -o json' % component)
+    if r:
+        items = []
+        for i in r:
+            data = json.loads(i['stdout'])
+            if 'items' in data:
+                items.extend(data['items'])
+        return items
+    else:
+        assert False, "unable to get %s" % component
```

Comment:
How about move this line to above and drop the if ... else ... branch. Just following the code style using in other functions.